flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionTest.java

```diff
@@ -82,202 +82,12 @@
 	private final TestingComponentMainThreadExecutor testMainThreadUtil =
 		EXECUTOR_RESOURCE.getComponentMainThreadTestExecutor();
 
-	/**
-	 * Tests that slots are released if we cannot assign the allocated resource to the
-	 * Execution.
-	 */
-	@Test
-	public void testSlotReleaseOnFailedResourceAssignment() throws Exception {
```

Comment:
Removed cases which test the legacy scheduling code paths, e.g. allocateResourcesForExecution()
ExecutionSlotAllocators have taken over slot allocation and there are tests for them.

Reply:
`testSlotReleaseOnFailedResourceAssignment` should indeed be obsolete.